### PR TITLE
Components update

### PR DESCRIPTION
**Pull request by nshahan**
* Glyph: Add baseline attribute.
 * Material Expansionpanel:
   * Allow programmatic `expand` and `collapse` requests even if the panel is
     `disabled`.
   * Change `preserveWhitespace` default to false.
   * Add `enterAccepts` flag.
 * Material Input:
   * Show counter regardless of focus.
   * Add error message to `MaterialPercentInputDirective`.
 * Material List: Allow pointer events when disabled.
 * Material Popup: Fix positioning when `matchMinSourceWidth` is true.
 * Material Ripple: Show a centered ripple on keypress.
 * Material Select:
   * Only display underline when border is also set.
   * Add `useCheckMarks` to use check marks instead of checkboxes.
 * Material Tooltips: Resize to fit contents.
 * Material Yes/No Buttons: Add `enterAccepts` flag.
 * Deprecate LazyStreamController and migrate to StreamController.
 * Migrate use of LazyEventController to StreamController.
 * Documentation fixes.
 * Strong mode fixes.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-version: 0.5.0
+version: 0.5.0+1
 description: >
   The official Material Design components for AngularDart. Used at Google
   in production apps.
 homepage: https://github.com/dart-lang/angular_components
 author: Dart Team <misc@dartlang.org>
 environment:
-  sdk: '>=1.23.0 <2.0.0'
+  sdk: '>=1.23.0-dev.11.11 <2.0.0'
```
Comment (kevmoo) [inline pubspec.yaml]: Remove this, right?

File: pubspec.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 name: angular_components
-version: 0.5.0
+version: 0.5.0+1
```
Comment (kevmoo) [inline pubspec.yaml]: New features? No breaks? Should be `0.5.1`

File: pubspec.yaml (hunk 1 above)
Comment (nshahan) [inline pubspec.yaml]: Oops yeah. I added that in so I could build on my desktop. I will remove.

File: pubspec.yaml (hunk 2 above)
Comment (nshahan) [inline pubspec.yaml]: Done.

**Comment by nshahan** [comment]
Yes, I verified with the angular_components_example. I will make a small update there to demo the new checkmark option for material select and bump the dependency.

**Pull request merged**